docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -37,7 +37,7 @@ Make sure you have `prepare` goal of `quarkus-maven-plugin` enabled in your `pom
             <executions>
```

Comment:
On line 28:
Make sure you have `prepare` goal of `quarkus-maven-plugin` enabled in your `pom.xml`:
 => 
Make sure you have `generate-code` goal of `quarkus-maven-plugin` enabled in your `pom.xml`:

Reply:
Alternatively: `generate-code` and `generate-code-tests` goals

Reply:
done